ts/dx.all.d.ts

```diff
@@ -3378,10 +3378,6 @@ declare module DevExpress.ui {
         field(): DevExpress.core.dxElement;
         /** @name dxDropDownEditor.open() */
         open(): void;
-        /** @name dxDropDownEditor.reset() */
-        reset(): void;
-        /** @name Component.resetOption(optionName) */
-        resetOption(optionName: string): void;
```

Comment:
@IlyaKhD  have any ideas why the resetOption method is gone from TS definitions as well?

Reply:
It was a bug in devextreme-internal-tools.